### PR TITLE
Entry point and stubs for state migration in v0.9.

### PR DESCRIPTION
**Pull request by anorth**
We are considering performing another small upgrade on the release/v0.9 branch, ahead of the larger upgrade/migration to v2. We're still working out exactly what we can/should include, but probably:
- repair miners broken by #1100 
- reduce some Window PoSt penalties (spec coming in another issue)
- update reward baseline parameters (previously implemented in /v2)

This PR adds the entry point and stubs for the state migration part.

FYI @acruikshank we'll probably put the code for #1177 here.

### CODE REVIEW

File: actors/migration/nv3/migrators.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package nv3
```
Comment (ZenGround0) [inline actors/migration/nv3/migrators.go:1]: I get 3 for third in place, what does nv mean?

File: actors/migration/nv3/miner.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	"context"
+
+	cid "github.com/ipfs/go-cid"
+	cbor "github.com/ipfs/go-ipld-cbor"
+
+	miner "github.com/filecoin-project/specs-actors/actors/builtin/miner"
+)
+
+type minerMigrator struct {
+}
```
Comment (ZenGround0) [inline actors/migration/nv3/miner.go:13]: TODO: we'll want to inject a way to set the power actor in here.

File: actors/migration/nv3/top.go [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+			CallSeqNum: actorIn.CallSeqNum,
+			Balance:    actorIn.Balance,
+		}
+
+		return tree.SetActor(addr, &actorOut)
+	}); err != nil {
+		return cid.Undef, err
+	}
+
+	return tree.Flush()
```
Comment (ZenGround0) [inline actors/migration/nv3/top.go:60]: This looks good

**Review by ZenGround0 — APPROVED** [review]

File: actors/migration/nv3/migrators.go (hunk 1 above)
Comment (anorth) [reply actors/migration/nv3/migrators.go:1]: "network version" (to distinguish from actors version)

**Pull request merged**